deploy/backplane/lpsre/10-lpsre-addon-operator-olm-apis.ClusterRole.yml

```diff
@@ -0,0 +1,18 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Can this be narrowed down to a specific namespace? Do you really need cluster-wide permissions?